package.yaml

```diff
@@ -30,19 +30,27 @@ dependencies:
 - base >= 4.7 && < 5
 - aeson
 - random
+- Cabal
```

Comment:
I don't think cabal should be dependency ? Similar for other packages - why are they required ?